.golangci.yml

```diff
@@ -0,0 +1,26 @@
+version: "2"
```

Comment:
Can you please keep this file aligned with the configuration from my other repositories, e.g. https://github.com/ricoberger/echoserver/blob/main/.golangci.yaml